### DISCUSSION TITLE
Image is missing src property.

### DISCUSSION THREAD

**Discussion by abdulbasit010** [vercel/next.js#19946] [Help]
getting an error Image is missing src property while fetching images urls from server.
Even i am passing src to Image component but still gets this error.
Here is my code.
```
<Image
    className="mw-100"
    alt={"alt"}
    src={thumbs[i]}
    layout="fill" 
 />
```

![image](https://user-images.githubusercontent.com/50239329/101452296-4e130100-394f-11eb-80b4-62db7b0428a5.png)

**Comment by jamesmosier** [comment]
Is it possible that `thumbs[i]` is missing a value?
